build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -151,6 +151,7 @@ define("$bootstrapModuleName", ["$moduleName", "dart_sdk"], function(app, dart_s
   dart_sdk.dart.setStartAsyncSynchronously(true);
   dart_sdk._isolate_helper.startRootIsolate(() => {}, []);
 $_initializeTools
```

Comment:
Want to fix the indentation while you're in here?

Reply:
Done.